src/main/webapp/features/iq/charting/iqChart.js

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc. All rights reserved.
```

Comment:
This has the wrong date and it will fail the license header check.

Reply:
Will change to 2016 year